README.md

```diff
@@ -82,29 +82,20 @@ Arguments:
   <project>
 
 Options:
-  --skip-backup                                Disables backing up the project. This is
-                                               not recommended unless the project is in
-                                               source control since this tool will make
-                                               large changes to both the project and
-                                               source files.
-  --extension <extension>                      Specifies a .NET Upgrade Assistant
-                                               extension package to include. This could
-                                               be an ExtensionManifest.json file, a
-                                               directory containing an
-                                               ExtensionManifest.json file, or a zip
-                                               archive containing an extension. This
-                                               option can be specified multiple times.
-  -e, --entry-point <entry-point>              Provides the entry-point project to start
-                                               the upgrade process. This may include
-                                               globbing patterns such as '*' for match.
-  -v, --verbose                                Enable verbose diagnostics
-  --non-interactive                            Automatically select each first option in
-                                               non-interactive mode.
-  --non-interactive-wait                       Wait the supplied seconds before moving
-  <non-interactive-wait>                       on to the next option in non-interactive
-                                               mode.
-  --version                                    Show version information
-  -?, -h, --help                               Show help and usage information
+  --skip-backup                                  Disables backing up the project. This is not recommended unless the project is in source control since this tool
```

Comment:
Another nit-pick - it might be good to update this without making the lines quite as long. This will look fine in a maximized window but may require scrolling if the person reading the docs doesn't have their browser maximized.